src/sickle.ts

```diff
@@ -223,6 +232,9 @@ class Annotator {
     if (/^\/\*\*/.test(comment) && /\*\/$/.test(comment)) {
       return comment.slice(3, comment.length - 2);
     }
+    if (/^\/\/\//.test(comment)) {
```

Comment:
I am being super pedantic but according to http://usejsdoc.org/about-getting-started.html jsdoc must start with `/**`. Maybe rename method to `getComment`